pkg/feature/templates/serverless/serving-install/knative-serving.tmpl

```diff
@@ -0,0 +1,21 @@
+apiVersion: operator.knative.dev/v1beta1
+kind: KnativeServing
+metadata:
+  name: {{ .Serving.Name }}
+  namespace: {{ .Serving.Namespace }}
+  annotations:
+    serverless.openshift.io/default-enable-http2: "true"
+spec:
+  deployments:
+    - annotations:
+        sidecar.istio.io/inject: "true"
+        sidecar.istio.io/rewriteAppHTTPProbers: "true"
+      name: activator
+    - annotations:
+        sidecar.istio.io/inject: "true"
+        sidecar.istio.io/rewriteAppHTTPProbers: "true"
+      name: autoscaler
+  ingress:
+    istio:
+      enabled: true
+      # local-gateway.{{ .Serving.Namespace }}.knative-local-gateway: "{{ .Serving.LocalGatewayServiceName }}.{{ .Mesh.Namespace }}.svc.cluster.local"
```

Comment:
@RetoCode May this template make sense to you?

Reply:
I think so yes, but you should definitely test that. "officially" we only support changing the `Mesh.Namespace`.

Reply:
@ReToCode you mean that the only supported installation is when in `knative-serving` namespace?

Reply:
Yes, so for the pattern, just remove the other params and just use `Mesh.Namespace`. Then that matches with what we document in our Serverless docs.

Reply:
Ok, thanks for the feedback. I'll fix.